RESTResource/RESTResource.js

```diff
@@ -768,11 +783,7 @@ export default class RESTResource {
               dispatch(this.actions.offsetFetchSuccess(meta, data));
             });
           }
-        }).catch((err) => {
-          dispatch(this.actions.fetchError({
-            message: `Unexpected fetch error ${err}`,
-          }));
-        });
+        }).catch((reason) => this.fetchAbortOrError(reason, dispatch));
```

Comment:
I would suggest naming like below. As `fetch` at the beginning is kind of naming for making the request. What do you think?
```suggestion
        }).catch((reason) => this.handleFetchOrAbortError(reason, dispatch));
```

Reply:
Good thinking @VictorSoroka1!